source/sass/mixins/_logical-properties.scss

```diff
@@ -1,21 +1,21 @@
 @import "types";
 @import "validation";
 
-@mixin _when-left-to-right {
+@mixin when-left-to-right {
```

Comment:
Ok to postpone, but we should move these elsewhere and add tests.

Reply:
👍 Logical-property-related mixins definitely need a refactor.